addons/emoji-picker/userstyle.css

```diff
@@ -87,12 +87,13 @@
 }
 .sa-emoji-picker-see-more {
   width: 100%;
-  margin-top: 5px;
+  margin-top: 5px !important;
+  margin-bottom: 5px !important;
 }
 .sa-emoji-button-r2 .sa-emoji-picker-see-more {
   width: calc(100% - 2px);
   margin-bottom: 7px;
```

Comment:
Hmm If you test it out, all you need to do it play with this value instead of adding a whole new property for `margin-top` and `margin-bottom`. In my opinion, the top margin doesn't really need to be changed, all you really need is to make the box a tad bit bigger to accomodate the button, which can be done without adding new lines to the file.

Reply:
That way, you wouldn't need the `!important` property, either. Since none of it gets overridden by vanilla scratch, in my testing.